viscy/representation/embedding_writer.py

```diff
@@ -93,8 +94,13 @@ def write_on_epoch_end(
         ultrack_indices = pd.concat([pd.DataFrame(p["index"]) for p in predictions])
         _logger.info(f"Computing UMAP embeddings for {len(features)} samples.")
         _, umap = _fit_transform_umap(features, n_components=2, normalize=True)
+        _, phate = _fit_transform_phate(
+            features, n_components=2, knn=5, decay=40, n_jobs=-1
+        )
         ultrack_indices["UMAP1"] = umap[:, 0]
         ultrack_indices["UMAP2"] = umap[:, 1]
+        ultrack_indices["PHATE1"] = phate[:, 0]
+        ultrack_indices["PHATE2"] = phate[:, 1]
```

Comment:
@Soorya19Pradeep See these changes.

Reply:
(Replying to https://github.com/mehta-lab/VisCy/pull/210#issuecomment-2543392649): do you have issues using this feature?

Reply:
What we need to do is essentially is to expose some of the other Phate parameters and put them in the config. It might be better to expose it as a callable. Especially if we want to change the KNN clustering or similar parameters